src/test/regress/sql/alter_table_set_access_method.sql

```diff
@@ -274,3 +274,19 @@ COMMIT;
 SET client_min_messages TO WARNING;
 DROP SCHEMA alter_table_set_access_method CASCADE;
 SELECT 1 FROM master_remove_node('localhost', :master_port);
+
+--test failure when a view is being tried to be altered to columnar
+CREATE SCHEMA view_access_method_test;
+SET search_path TO view_access_method_test;
+
```

Comment:
No need to create a new schema for this IMO. We usually have a single schema, if we do not directly test some case that involves multiple schemas.

I suggest you (1) remove the first 3 lines you added, (2) move your tests before we drop `alter_table_set_access_method` schema, (3) do not drop any schema as we already do it in this file.

Reply:
Fixed you as you suggested